.changeset/tasty-eagles-draw.md

```diff
@@ -0,0 +1,30 @@
+---
+"@spectrum-css/link": major
+---
+
+# Link S2 migration
+
+The link component is now updated with S2 specs, colors, and typography.There is a new inline variant which uses the `.spectrum-Link--inline` class and the default variant is the standalone variant.
+
+**Notes**
+- Quiet styling does not apply to the inline variant for accessiblity reasons.
```

Comment:
```suggestion
- Quiet styling does not apply to the inline variant for accessiblity reasons.
```

Any chance we can expand on those reasons a bit just to make sure we have them documented here for the changelog?